praw/objector.py

```diff
@@ -1,10 +1,29 @@
 """Provides the Objector class."""
+import re
+
 from .exceptions import APIException
 
 
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def camel_to_snake(name):
+        """Convert camelCase string to snake_case."""
```

Comment:
Since this isn't modifying the passed in value, consider "Return `name` converted from camelCase to snake_case."